notebooks/contrib-dev/Alcohol_Pharmacokinetics.ipynb

```diff
@@ -0,0 +1,1075 @@
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 4,
+      "metadata": {
+        "id": "fu_R4FsO6ej4"
+      },
+      "outputs": [],
+      "source": [
+        "def f(t, y):\n",
+        "    \"\"\"RHS of differential equation for reaction kinetics\n",
+        "    Args:\n",
+        "        t: time\n",
+        "        y: values for differential questions, [CA, CB, BAC]\n",
+        "    Returns:\n",
+        "        dydt: first derivation of y w.r.t. t\n",
+        "    \"\"\"\n",
+        "\n",
+        "    # Unpack current values of y\n",
+        "    C0, CA, CB, BAC = y\n",
+        "\n",
+        "    # dC0 rate constant.\n",
+        "    # if statement added so models don't go below 0\n",
+        "    if C0 < 0:\n",
+        "        k0 = 0\n",
+        "    else:\n",
+        "        k0 = 10  # 1/h\n",
+        "\n",
+        "    # dCa rate constant\n",
+        "    k1 = 10  # 1/h\n",
+        "\n",
+        "    # dCb and dBAC rate constant\n",
+        "    # if statement added so models don't go below 0\n",
+        "    if CB < 0:\n",
+        "        k2 = 0\n",
+        "    else:\n",
+        "        k2 = 1.92  # g/hL\n",
+        "\n",
+        "    # Store equations as a list in the variable dydt\n",
+        "\n",
+        "    # Add your solution here\n",
+        "    ### BEGIN SOLUTION\n",
+        "    dC0 = -k0 * C0\n",
+        "    dCA = (k0 * C0) - k1 * CA\n",
+        "    dCB = k1 * CA - k2\n",
+        "    # BAC is in g/dL, CB is in g/L, so divided by 10\n",
+        "    dBAC = dCB / 10  # g/100mL\n",
+        "    ### END SOLUTION\n",
+        "    dydt = [dC0, dCA, dCB, dBAC]\n",
+        "\n",
+        "    return dydt"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "zUVAuNrrEdgP"
+      },
+      "source": [
+        "\n",
+        "Discuss in 3-5 sentences what theses ODEs say about the Pharmacokinetics of alcohol.\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "rFj5qSrOM1H9"
+      },
+      "source": [
+        "**Discussion:**"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "-XjcPqq0K6K0"
+      },
+      "source": [
+        "### 1C. BAC Modeling: Sobering up\n",
+        "\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "tTT4vyOBN-AE"
+      },
+      "source": [
+        "This function solves the ODE for dCa/dt, dCb/dt, and dBAC/dt that you made in part 1B.\n",
+        "\n",
+        "**Its important to note that this ODE assumes that a person of \"m\" weight  drinks \"n\" number of drinks on a moderatley full stomach at a given rate then stops.**"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 5,
+      "metadata": {
+        "id": "HjigLETp9QIi"
+      },
+      "outputs": [],
+      "source": [
+        "def SolveODE(CT, tmax, f):\n",
```

Comment:
Change function name to solve_ode. Best practice in Python is to not capitalize function names (that often means classes).